odoo/addons/test_new_api/tests/test_schema.py

```diff
@@ -577,3 +577,36 @@ def test_10_many2many(self):
             (field.relation, field.column1, model._table, 'id', 'CASCADE'),
             (field.relation, field.column2, comodel._table, 'id', 'CASCADE'),
         ])
+
+    def test_20_unique_indexes(self):
+        """ Test uniqueness of indexes:
+        - test_new_api.order.line_short_field_name
+        - test_new_api.order.line.short_field_name
+        """
+        indexname = (
+            "test_new_api_order__line_short_field_name_index",
+            "test_new_api_order_line__short_field_name_index"
+        )
+        self.env.cr.execute("SELECT tablename FROM pg_indexes WHERE indexname IN %s", [indexname])
+        tables = {table for table, in self.env.cr.fetchall()}
+        self.assertEqual(tables, {'test_new_api_order', 'test_new_api_order_line'})
```

Comment:
I understand what this does, but the way it's tested is really the reverse of what you'd expect.

Reply:
You are right, I changed both tests (the second didn't show the issue either).